esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/jasmin.cylc

```diff
@@ -25,12 +29,17 @@
         [[[directives]]]
             --mem = 2G
 
-    [[process<fast>]]
-        execution time limit = PT10M
+    # Resources for recipes that need more than the default.
+    # Variable (fast, medium) must be consistent with flow.cylc.
+    # Comment indicates example recorded usage on Jasmin.
```

Comment:
Should `Jasmin` be `JASMIN`? 🤔

Reply:
Fixed in [2385db2](https://github.com/ESMValGroup/ESMValTool/pull/3562/commits/2385db208e893d4284426a58c94fd51ddf16f5d1)